### PR TITLE
Add comment to uncommitted PRs

### PR DESCRIPTION
**Pull request by sandersn**
I haven't spent any time editing the message, since I'm not even sure I've got the structure of the code right.

There are also a couple of API calls that should probably be de-duped now that they're happening 3 times overall.

### CODE REVIEW

File: src/checks/addCommentToUncommittedPRs.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  }
+  else {
+    const isSuggestion = relatedIssues.some(issue => issue.labels?.find(l => l.name === "Suggestion"))
+    const isCommitted = relatedIssues.some(issue => issue.labels?.find(l => l.name === "Committed" || l.name === "Experience Enhancement" || l.name === "help wanted"))
+    if (isSuggestion && !isCommitted && !(await isMemberOfTSTeam(pull_request.user.login, api, logger))) {
+      const comments = (await api.issues.listComments(thisIssue)).data
+      if (!comments || !comments.find(c => c.body.startsWith('The typescript team has'))) {
+        await api.issues.createComment({
+          ...thisIssue,
+          body: `The typescript team hasn't accepted the linked issue #${relatedIssues[0].number}. This makes it less likely that we'll review or accept this PR. Try to get the originating issue accepted.`
```
Comment (orta) [inline src/checks/addCommentToUncommittedPRs.ts:41]:
```suggestion
          body: `The TypeScript team hasn't accepted the linked issue #${relatedIssues[0].number}. This makes it less likely that we'll review or accept this PR. Try to get the originating issue accepted.`
```

File: src/checks/addCommentToUncommittedPRs.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        body: "The PR doesn't have any linked issues. Please open an issue that references this PR. From there we can discuss and prioritise."
+      })
+    }
+  }
+  else {
+    const isSuggestion = relatedIssues.some(issue => issue.labels?.find(l => l.name === "Suggestion"))
+    const isCommitted = relatedIssues.some(issue => issue.labels?.find(l => l.name === "Committed" || l.name === "Experience Enhancement" || l.name === "help wanted"))
+    if (isSuggestion && !isCommitted && !(await isMemberOfTSTeam(pull_request.user.login, api, logger))) {
+      const comments = (await api.issues.listComments(thisIssue)).data
+      if (!comments || !comments.find(c => c.body.startsWith('The typescript team has'))) {
```
Comment (orta) [inline src/checks/addCommentToUncommittedPRs.ts:38]:
```suggestion
      if (!comments || !comments.find(c => c.body.startsWith('The TypeScript team has'))) {
```

**Comment by orta** [comment]
Looks good

**Comment by sandersn** [comment]
I may come back through with another editing pass later today. I'll be sure to keep TypeScript's bicaps intact. =)

**Comment by sandersn** [comment]
It's leaving comments now.

**Pull request merged**